index.js

```diff
@@ -39,7 +40,12 @@ async function run() {
         try {
             await exec.exec(command);
         } catch (err) {
-            core.setFailed('The ZAP Baseline scan has failed, starting to analyze the alerts. err: ' + err.toString());
+            if (failAction == 'true') {
```

Comment:
case ignore match? Should we report if its not something we expect?

Reply:
Fixed